### PR TITLE
Create OSPO

### PR DESCRIPTION
**Pull request by clausmullie**

### CODE REVIEW

**Review by ericherman — APPROVED** [review]
The top does a pretty good job of scoping it to codebase work.
I'm fine with it, as-is if we want a super-soft launch. But rather than [add it to _config.yml](https://github.com/publiccodenet/publiccode.net/pull/253/commits/c0c945d247f4f5b3cd00c2afec046613ac887fcd#diff-ecec67b0e1d7e17a83587c6d27b6baaaa133f42482b07bd3685c77f34b62d883R16), I'd rather we link it from another page.

**Comment by Ainali** [comment]
> But rather than [add it to _config.yml](https://github.com/publiccodenet/publiccode.net/pull/253/commits/c0c945d247f4f5b3cd00c2afec046613ac887fcd#diff-ecec67b0e1d7e17a83587c6d27b6baaaa133f42482b07bd3685c77f34b62d883R16), I'd rather we link it from another page.

I totally agree! I did this as a fallback and also as way to be able to see how it looked when served.

File: OSPO.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+type: Resource
+---
+
+# Government Open Source Programme Offices: let's work together
```
Comment (ElenaFdR) [inline OSPO.md:5]:
```suggestion
# Government OSPOs: let's work together
```

File: OSPO.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* [clarify and improve governance around the codebase](activities/supporting-codebase-governance/index.md)
+* [develop good working practices that make collaboration easier and improve the quality of the codebase](https://standard.publiccode.net)
+* [work on the communication, culture change and organizational values that come with collaboration in the open](https://publiccode.net/codebase-stewardship)
+
+Email us at <info@publiccode.net> to start the discussion.
+
+## Getting started with a government OSPO
+
+If you're looking for more general OSPO advice or government digital transformation tips, we recommend:
```
Comment (ElenaFdR) [inline OSPO.md:26]:
```suggestion
If you're looking for more general OSPO advice or government digital transformation guidance, we recommend:
```

**Review by ElenaFdR — COMMENTED** [review]
Small changes

**Pull request merged**